synapse/lib/stormlib/gen.py

```diff
@@ -425,6 +435,36 @@ class LibGen(s_stormtypes.Lib):
             [ geo:place=(gen, name, $geoname) :name=$geoname ]
             return($node)
         }
+
+        function __getCryptoNodeBySha256(sha256) {
+            // Try to lift crypto:x509:cert by sha256
+            crypto:x509:cert:sha256=$sha256
+            return($node)
+
+            // Try to lift crypto:x509:cert by file
+            crypto:x509:cert:file = { file:bytes:sha256=$sha256 }
```

Comment:
I think you were planning on changing this, but the file:bytes query could return >1.  You should just be able to do file:bytes:sha256=$sha256 -> crypto:x509:cert:file